api-report/task-manager.api.md

```diff
@@ -18,9 +18,10 @@ import { SharedObject } from '@fluidframework/shared-object-base';
 // @public
 export interface ITaskManager extends ISharedObject<ITaskManagerEvents> {
     abandon(taskId: string): void;
-    haveTaskLock(taskId: string): boolean;
-    lockTask(taskId: string): Promise<void>;
+    assignedTask(taskId: string): boolean;
```

Comment:
The name here is a bit ambiguous. I was not sure if this is (get)AssignedTask or (has)AssignedTask until I looked at the return value.

Reply:
I named it `assignedTask` instead of `isAssignedTask` to better match `queued()`. However, I definitely see how this can be confusing. Do you think I should switch it to `isAssignedTask`?

Reply:
I think it's good to be consistent with "is" between assignedTask and queued -- I'm fine with or without "is" personally.

Reply:
maybe`taskAssigned` would line it up little bit better. Or just `assigned` if you want to make it truly consistent with `queued`. Both are boolean flags on Task itself.

Reply:
unresolving so you can see my response. Feel free to make your own conclusions here.

Reply:
I think `assigned` makes the most sense, especially since I just added `subscribed`